### ISSUE TITLE
[Feature]: Show original total and discounted amount in the sales register summary

### ISSUE THREAD

**Issue by objecttothis** [opensourcepos/opensourcepos#4118]
### Type of Feature

✨ New Feature

### OpensourcePOS Version

opensourcepos 3.3.9

### Description

Currently, when an item is discounted, the summary shows the subtotal with the discounted amount, then the total is repeated with that amount.  I don't think this is how it should be.  I think the Subtotal should be the full price amount, then we need a line below showing total discount, then the total below that should be the subtotal, minus the total discount.  As is, I think subtotal and total will always just be redundant.

![image](https://github.com/user-attachments/assets/31af2302-b2af-41f2-88e8-3a01c536a6ac)

Since 3.4.0 is a release focused on just getting current behavior released in CI4, this should probably go into 3.4.1.

### Verify you searched open requests in OpensourcePOS

- [X] I agree I have searched Open Requests

**Comment by jekkos** [comment]
Its probably related to decimal.precision and rounding behavior, looking at the numbers, they do not seem to be off by a.lot

**Comment by odiea** [comment]
@jekkos I believe he is looking for something like the following. Correct me if I am wrong.
![2024-12-23 13_27_12-Window](https://github.com/user-attachments/assets/aa939164-c6a0-4b62-a758-527b4f439712)

**Comment by jekkos** [comment]
Right, comments were meant for another issue, thanks for the screenshot here @odiea
